packages/bpk-component-icon/README.md

```diff
@@ -30,7 +30,7 @@ export default () => (
 @import '~bpk-mixins';
 
 .abc-icon__flight {
-  fill: $bpk-color-white;
+  fill: currentColor; // https://www.w3schools.com/colors/colors_currentcolor.asp
```

Comment:
Can we link to [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentColor) instead of W3Schools? Or mabye [CSS Tricks](https://css-tricks.com/currentcolor/)

Reply:
Makes sense, added both :-P

Reply:
I think just having one might be the best here as it looks to be a bit messy having both and overkill - I am in favour of CSS tricks. Can we just revert to one?

Reply:
You're right, nicer with just one 😅 
Changed to CSS tricks.